libnmstate/ifaces/ovs.py

```diff
@@ -184,6 +187,32 @@ def parent(self, value):
     def need_parent(self):
         return True
 
+    @property
+    def is_patch_port(self):
+        return self._patch_config and self._patch_config.get(
+            OVSInterface.Patch.PEER
+        )
+
+    def pre_edit_validation_and_cleanup(self):
+        super().pre_edit_validation_and_cleanup()
```

Comment:
Need also validate the existence of patch peer, should raise exception:
 * Peer not exists.
 * Peer down or absent.

Reply:
AFAICS, this is valid. OVS Patch can be created on NM with a peer that does not exist or is down.

Reply:
Done, thanks!